docs/clients/cocoa/advanced.rst

```diff
@@ -13,7 +13,7 @@ Sending a basic message (no stacktrace) can be done with `captureMessage`.
 
     SentryClient.shared?.captureMessage("TEST 1 2 3", level: .Debug)
 
-If more detailed information is required, `Event` has a large constructor
```

Comment:
Let me know if this is a thing. My googling yielded a bunch of off-topic results so it doesn't seem like a standard term.